src/celeritas/io/ImportAtomicRelaxation.hh

```diff
@@ -17,15 +17,15 @@ namespace celeritas
  */
 struct ImportAtomicTransition
 {
-    int    initial_shell; //!< Originating shell designator
-    int    auger_shell;   //!< Auger shell designator
-    double probability;   //!< Transition probability
-    double energy;        //!< Transition energy [MeV]
+    int    initial_shell{}; //!< Originating shell designator
+    int    auger_shell{};   //!< Auger shell designator
+    double probability{};   //!< Transition probability
+    double energy{};        //!< Transition energy [MeV]
 };
 
 struct ImportAtomicSubshell
 {
-    int                                 designator; //!< Subshell designator
+    int                                 designator{}; //!< Subshell designator
     std::vector<ImportAtomicTransition> fluor;      //!< Radiative transitions
     std::vector<ImportAtomicTransition> auger; //!< Non-radiative transitions
```

Comment:
We load both transitions by checking one single boolean defined in `GeantPhysicsOptions`, while G4 has an individual setters for each. Do you think is it worth making a note of this caveat in the class description for future reference, in case we end up having to come back here to enable/disable them individually?

Reply:
Where's the best place to note this? I don't think our atomic relaxation class in general supports turning on only one or the other. In fact, Geant being Geant, do we know that the options work as advertised, and whether you can only do auger when fluor is enabled, etc.?

Reply:
Cool, I thought I remembered something like that but couldn't find it (maybe I was confused because we always read the auger data?)

Reply:
Yeah good point, we always read all the transition data (but only store it in the params if enabled).

Reply:
So maybe I'll make it an enum with three settings: no relaxation, no auger, relaxation? Any better name suggestions?

Reply:
Ohhh, I missed that too. Are we then cascading that down so we define `is_auger_enabled` based on fluor/auger bools instead of just the current `atomic_relaxation` one?

Reply:
Your updated suggestions sound good :)

Reply:
Ha cascading down the Auger cascade

Reply:
I've changed the behavior as discussed. There's a new EM parameter `auger`, and the presence of `atomic_relaxation` data should be a sufficient proxy for the `fluor` option.